src/System.Text.Encoding.CodePages/src/System/Text/EncodingTable.Data.cs

```diff
@@ -1250,6 +1250,7 @@ internal static partial class EncodingTable
             28599, // iso-8859-9
             28603, // iso-8859-13
             28605, // iso-8859-15
+            29001, // x-europa
```

Comment:
Found the issue, I think, but now that I have data files I need to change stuff in the tests too (can feed those in).
Ideally the goal is to make this a build target, yes (even if only on a specific one)?